bin/test-workflow/start

```diff
@@ -135,14 +146,23 @@ test_app_workflow="
 ./8_app_verify_authentication.sh"
 
 if [[ "$CONJUR_OSS_HELM_INSTALLED" == "true" ]]; then
+  if [[ "$CONJUR_PLATFORM" == "oc" || "$APP_PLATFORM" == "oc" ]]; then
+    ./platform_login.sh
```

Comment:
Does this need a `-o` option to specify openshift here? Also does this need to be pulled out of the conditional since `platform_login.sh` is used for GKE as well as OpenShift?

Reply:
yes, it is clearer to have the explicit option. Added it.